custom_components/plugwise_usb/coordinator.py

```
@@ -87,7 +87,7 @@ async def async_node_update(self) -> dict[NodeFeature, Any]:
             and self.node.initialized
             and not states[NodeFeature.AVAILABLE].state
         ):
-            raise UpdateFailed("Device is not responding")
+            raise UpdateFailed(f"Device '{self.node._mac_in_str}'is (temporarily) not available")
```

Ruff (0.12.2)

Private member accessed: `_mac_in_str`

(SLF001)